bcda/api/v1/api_test.go

```diff
@@ -126,17 +126,18 @@ func (s *APITestSuite) TestBulkEOBRequestNoQueue() {
 }
 
 func (s *APITestSuite) TestBulkPatientRequest() {
-	since := "2020-02-13T08:00:00.000-05:00"
 	bulkPatientRequestHelper("Patient", "", s)
 	s.TearDownTest()
 	s.SetupTest()
 	bulkPatientRequestHelper("Group/all", "", s)
 	s.TearDownTest()
 	s.SetupTest()
-	bulkPatientRequestHelper("Patient", since, s)
-	s.TearDownTest()
-	s.SetupTest()
-	bulkPatientRequestHelper("Group/all", since, s)
+	for _, since := range []string{"2020-02-13T08:00:00.000-05:00", "2020-02-13T08:00:00.000+05:00"} {
+		bulkPatientRequestHelper("Patient", since, s)
+		s.TearDownTest()
+		s.SetupTest()
+		bulkPatientRequestHelper("Group/all", since, s)
```

Comment:
Do we need to call `TearDownTest` and `SetupTest` explicitly after the second test in the loop so that the framework is normalized before the next loop iteration?